controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -86,9 +90,17 @@ func configureServiceMeshFeatures() feature.FeaturesProvider {
 			}
 		}
 
+		// AuthRefs takes in obtained audiences slice and this can be used to overwrite default if different.
+		var audiences []string
+		if isDefaultAudiences(handler.ServiceMesh.Auth.Audiences) {
+			audiences = r.Audiences
+		} else {
+			audiences = *handler.ServiceMesh.Auth.Audiences
+		}
```

Comment:
I would extract it to a func called `definedAudiencesOrDefault` and inline down below. That would make it more readable in my eyes.

Reply:
[1374f2a](https://github.com/opendatahub-io/opendatahub-operator/pull/947/commits/1374f2a24845425f7001d66c0938b062d2c37e5b) done